app/src/main/kotlin/no/nav/tiltakspenger/vedtak/repository/felles/PeriodeDbJson.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.tiltakspenger.vedtak.repository.felles
+
+import no.nav.tiltakspenger.libs.periodisering.Periode
+import java.time.LocalDate
+
+/**
+ * Skal kun brukes i route-laget. Dersom du trenger den til andre ser/des, bør den flyttes til common-lib.
+ */
+data class PeriodeDbJson(
```

Comment:
Hva står Db for her?

Reply:
Hvis det står for database, som jo gir mening, i og med at den ligger under repository-mappa, så er kanskje kommentaren her litt feil?

Reply:
Godt observert! Skal stå databaselaget. Den skal _ikke_ brukes i route laget!